.github/workflows/steelthread.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   end2endtest:
-    timeout-minutes: 15
+    timeout-minutes: 20
```

Comment:
Under what scenarios does this take more than 15 min?

Reply:
should never happen, it needs the timeout-minutes field to use the action module